py_gen/codegen.py

```diff
@@ -72,9 +72,11 @@ def generate_pyname(ofclass):
 
 # Create intermediate representation, extended from the LOXI IR
 def build_ofclasses(version):
-    modules = defaultdict(list)
+    modules = OrderedDict()
     for ofclass in loxi_globals.ir[version].classes:
         module_name, ofclass.pyname = generate_pyname(ofclass)
+        if module_name not in modules:
```

Comment:
this works too, of course!

Reply:
sorry, i didn't realize OrderedDefaultDict was in generic_utils, so i finally changed the code to use it